SQLiteTests/AggregateFunctionsTests.swift

```diff
@@ -1,5 +1,5 @@
 import XCTest
-import SQLite
+@testable import SQLite
```

Comment:
Good point.  I was doing this in another project so I just did the same here. Likely not needed at all.  Note, FYI, that I did change from including SQLite as part of the test target in Xcode, which is was previously.

I'll try without and push an update if it works ok, or add a comment here if not.